auth.go

```diff
@@ -87,78 +97,96 @@ func DecodeBase64Auth(auth AuthConfig) (string, string, error) {
 	decoded := make([]byte, decLen)
 	n, err := base64.StdEncoding.Decode(decoded, []byte(auth.Auth))
 	if err != nil {
-		return "", "", fmt.Errorf("error decoding auth from file: %w", err)
+		return "", "", fmt.Errorf("decode auth: %w", err)
 	}
 
-	if n > decLen {
-		return "", "", fmt.Errorf("decoded value is longer than expected length, expected: %d, actual: %d", decLen, n)
-	}
+	decoded = decoded[:n]
 
-	split := strings.SplitN(string(decoded), ":", 2)
-	if len(split) != 2 {
-		return "", "", errors.New("invalid auth string")
+	const authParts = 2
+	split := strings.SplitN(string(decoded), ":", authParts)
+	if len(split) != authParts {
+		return "", "", fmt.Errorf("invalid auth: %s", string(decoded))
```

Comment:
I don't think this should output the decoded output since it could have secrets in them, even though it may be malformed.